runtime/src/main/java/io/quarkiverse/resteasy/problem/openapi/HttpProblemSchema.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkiverse.resteasy.problem.openapi;
+
+import java.net.URI;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+@Schema(name = "HttpProblem", description = "HTTP Problem Response according to RFC9457 & RFC7807", additionalProperties = Schema.True.class)
+public class HttpProblemSchema {
```

Comment:
Maybe this class should be `HttpProblemResponse` instead of `Schema`?   Because on the other end it should decode to a domain object and the name `HttpProblemSchema` does not feel like a domain object but `HttpProblemResponse` describes what it actually is?

Reply:
Nevermind I see you override the name `name = "HttpProblem"`. Please ignore.